sdk/identity/azure-identity/azure/identity/aio/_credentials/azure_cli.py

```diff
@@ -32,9 +32,9 @@ class AzureCliCredential(AsyncContextManager):
 
     This requires previously logging in to Azure via "az login", and will use the CLI's currently logged in identity.
 
-    :keyword List[str] additionally_allowed_tenants: Optional additional tenant ids for which the credential
-        may acquire tokens. Add the wildcard value "*" to allow the credential to acquire tokens for
-        any tenant the application is installed.
+    :keyword List[str] additionally_allowed_tenants: Specifies tenants in addition to the specified for which the
+        credential may acquire tokens. Add the wildcard value "*" to allow the credential to acquire tokens for
+        any tenant the logged in account can access.
```

Comment:
Should we hold off on adding this parameter completely until we add the `tenant_id` parameter to the credential?

Reply:
I don't think we need. Because this is a keyword only argument, user can always specify it and w/o having the tenant_id support, it is always ignored. This is said, no matter we doc it or not, the parameter has no impact. Though we can hide it in doc if you want.

Reply:
I'd say hide the docs for now so users don't set this expecting it might have some impact.